src/main/kotlin/com/dansplugins/fiefs/Fiefs.kt

```diff
@@ -2,12 +2,17 @@ package com.dansplugins.fiefs
 
 import com.dansplugins.factionsystem.MedievalFactions
 import com.dansplugins.fiefs.command.fiefs.FiefsCommand
+import com.dansplugins.fiefs.fief.FiefFactory
+import com.dansplugins.fiefs.fief.FiefRepository
 import org.bstats.bukkit.Metrics
 import org.bukkit.plugin.java.JavaPlugin
 
 class Fiefs : JavaPlugin() {
     lateinit var medievalFactions: MedievalFactions
 
+    val fiefRepository = FiefRepository(this)
+    val fiefFactory = FiefFactory(this)
```

Comment:
Why do we need factory pattern for this?

Reply:
My intention was to make the Fief class instantiated only in the factory, so that was the only reason.

Reply:
What does that give us?

Reply:
I thought it might make sense to put all of the instantiations in one class to make it easy to find & change them. Is this just an unnecessary complication?

Reply:
I've removed the FiefFactory and opted for a less complicated instantiation method for now.